sql/postgres/convert.go

```diff
@@ -233,7 +233,22 @@ func parseColumn(s string) (*columnDesc, error) {
 	case TypeReal, TypeFloat4:
 		c.precision = 24
 	default:
-		c.typ = s
+		parts = strings.FieldsFunc(s, func(r rune) bool {
+			return r == '(' || r == ')'
+		})
```

Comment:
The input to the function is not `timestamp(4) with time zone` like it should, it is `timestamp with time zone(4)`. Now that I think about it, it would have made more sense to ensure the correct formatting is passed to the function instead of handling the bad one here. I'll try to change it.